pkg/scheduler/podeventshandler.go

```diff
@@ -220,8 +220,8 @@ func (sched *Scheduler) updatePod(oldObj, newObj interface{}) {
 		klog.InfoS("Failed to update pod in scheduler cache", "err", err)
```

Comment:
can we put `sched.commonCache.UpdatePod` and `podutil.FilteringUpdate(sched.assumedOrBoundPod, sched.triggerQueueOnAssumedOrBoundPodAdd, sched.triggerQueueOnAssumedOrBoundPodUpdate,
		sched.triggerQueueOnAssumedOrBoundPodDelete, oldPod, newPod)` together into one function ?
One is cache operations on pod update event and the other one is queue operations based on that, they are both cache related reactions.